shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/io/jackson/JsonMetaStoreEventSerDe.java

```diff
@@ -80,14 +80,15 @@ private void registerDeserializers(SimpleModule module) {
   @Override
   public <T extends SerializableListenerEvent> T unmarshal(byte[] payload) throws MetaException {
     try {
+      System.out.println(new String(payload));
```

Comment:
remove

Reply:
I need this for my testing. So, I will remove it before merging. ok ?

Reply:
sure